### ISSUE TITLE
MD014 in bash code

### ISSUE THREAD

**Issue by alanlivio** [DavidAnson/markdownlint#506]
From [VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint), I received the MD014 warning for the following explicit bash code.

```bash
$ docx_reviews_to_txt.py lorem_ipsum.docx
```

**Comment by DavidAnson** [comment]
MD014 warns about using the "$" character without showing command output. That seems to describe the example you show, so the warning seems correct?

https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md014

**Comment by alanlivio** [comment]
Yes @DavidAnson, you are right although I visually prefer with the $.

**Comment by nschonni** [comment]
You can always turn off the rule if you prefer to use the `$` style consistently
